app/views/visns/index.html.erb

```diff
@@ -5,6 +5,9 @@
   <%= render partial: "shared/messages", locals: { small_text: false } %>
   <section id="visns-map" class="margin-bottom-3">
     <h1 class="font-sans-2xl margin-top-8 margin-bottom-3 line-height-46px">VISNs</h1>
+    <div class="margin-bottom-2">
+      <span class="line-height-26">Veterans Integrated Services Networks are regional systems of care within the Veterans Health Administration.</span>
```

Comment:
Can we use a `p` element here with the class `usa-prose-body`? I don't think we'll need the `line-height-26`.

Reply:
The <p> tag cuts the text in to two lines.  <span with usa-prose-body ... keeps it all on one line...